metabulo/models.py

```diff
@@ -121,36 +146,50 @@ def read_csv_file(self, data):
         return data
 
 
-class CSVColumnType(db.Model):
-    type_ = db.Column(db.String, primary_key=True)
-    numeric = db.Column(db.Boolean(name='ck_numeric_flag'), nullable=False)
+class TransformType(db.Model):
+    name = db.Column(db.String, primary_key=True)
 
 
 event.listen(
-    CSVColumnType.__table__, 'after_create',
-    DDL("""
-INSERT INTO csv_column_type (type_, numeric) VALUES
-('object', 0), ('float64', 1)
-    """)
-)
+    TransformType.__table__, 'after_create',
+    DDL(f"""
+INSERT INTO transform_type (name) VALUES
+{','.join(["('" + t + "')" for t in transform.registry.keys()])}
+"""))
```

Comment:
Why does the registry need to be duplicated into the db?  Does this pr support adding new transforms after the initial create event?

Writing and maintaining migrations on stumpf wasn't too fun, so I'm curious if there are ways to avoid setting the project up to need those in the future.

Reply:
Oh yes.  For the moment, I'm avoiding doing migrations because we don't have any data we actually need to save.  Until we do, the migration procedure is delete the sqlite database and rerun `metabulo-create-tables`.

Reply:
> Why does the registry need to be duplicated into the db? Does this pr support adding new transforms after the initial create event?

It doesn't really.  At the moment, it's a sqlite version of ENUM.  There may be a need to store persistent information about the registry at some point.  If not, it should be easy enough to delete it.  (I already deleted the similar column type table that had existed.)